nrrd/tests/data/BallBinary30x30x30_nifti.nhdr

```diff
@@ -0,0 +1,14 @@
+NRRD0004
+# Complete NRRD file format specification at:
+# http://teem.sourceforge.net/nrrd/format.html
+type: short
+dimension: 3
+space: left-posterior-superior
+sizes: 30 30 30
+# byte skip: -1
```

Comment:
The `byte skip` has been intentionally commented out here to point out the error when nifti file is attempted to read from the beginning.